styleguide/index.html

```diff
@@ -8,5 +8,5 @@
   <body>
     <div id="app"></div>
     <footer>We got a footer!</footer>
-  <script type="text/javascript" src="build/bundle.422eb0d6.js"></script></body>
+  <script type="text/javascript" src="build/bundle.77113a7e.js"></script></body>
```

Comment:
This shouldn't be showing up. Make sure you've rebased your `.gitignore` and cleared your cache. https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore